plugins/nanospell/plugin.js

```diff
@@ -803,8 +803,7 @@
 				var combinedText = '',
 					block,
 					blockList = [],
-					iterator = range.createIterator(),
-					bookmarks = editor.getSelection().createBookmarks2(false);
```

Comment:
this was being done too far out, it was possible to then enter a shortcut case where we immediately start marking words.